### PR TITLE
EP-93 Cloudfront in front of handbook website - No access to S3

### PR DESCRIPTION
**Pull request by connoravo**
Cloudfront will deploy to dev but the FQDN dev.handbook.burendo.com will not be activated. This is reserved for prod.

### CODE REVIEW

File: .github/workflows/tf-merge.yml [hunk 1]
```diff
@@ -104,11 +104,11 @@ jobs:
...
-      - name: "Copy build output to S3"
-        id: copy-to-s3
-        run: aws s3 sync build s3://${ENV}.${BUCKET_SUFFIX} --delete
-        working-directory: burendo-handbook
-        continue-on-error: false
+      # - name: "Copy build output to S3"
+      #   id: copy-to-s3
+      #   run: aws s3 sync build s3://${ENV}.${BUCKET_SUFFIX} --delete
+      #   working-directory: burendo-handbook
+      #   continue-on-error: false
```
Comment (connoravo) [inline .github/workflows/tf-merge.yml:111]: Commented out for now as we don't want to deploy Prod

File: .github/workflows/tf-merge.yml [hunk 2]
```diff
@@ -59,17 +59,17 @@ jobs:
...
+      #   id: apply
+      #   run: |
+      #     terraform plan -no-color -out terraform.plan >> ${{ github.workspace }}/${{ github.run_id }}-${{ github.run_number }}.log
+      #     terraform apply -no-color -auto-approve terraform.plan >> ${{ github.workspace }}/${{ github.run_id }}-${{ github.run_number }}.log
+      #   working-directory: terraform
+      #   continue-on-error: false
+      #   env:
+      #     TF_WORKSPACE: default
+      #     TF_INPUT: false
+      #     AWS_DEFAULT_REGION: eu-west-2
```
Comment (connoravo) [inline .github/workflows/tf-merge.yml:72]: Commented out for now as we don't want to deploy Prod

File: .github/workflows/tf-merge.yml [hunk 3]
```diff
@@ -48,7 +48,7 @@ jobs:
         working-directory: terraform
```
Comment (ChrisScottThomas) [inline .github/workflows/tf-merge.yml:48]:
```suggestion
        working-directory: .
```

**Review by ChrisScottThomas — COMMENTED** [review]
One adjustment.

**Review by ChrisScottThomas — DISMISSED** [review]

**Review by steveburtonBUR — APPROVED** [review]

**Pull request merged**